src/modules/75-cf/components/TargetManagementDetailPageTemplate/TargetManagementDetailPageTemplate.tsx

```diff
@@ -113,8 +112,6 @@ const TargetManagementDetailPageTemplate: FC<TargetManagementDetailPageTemplateP
       {...detailPageTemplateProps}
     >
       <div className={cx(css.layout, ...modifiers)}>
```

Comment:
You can rip out the `modifiers` bit and kill likes 91-93 above as well. We no longer need to adjust the layout when GitSync is enabled.
```suggestion
      <div className={css.layout}>
```